### PR TITLE
complete ncnews-12

### PR DESCRIPTION
**Pull request by abdullah7x**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,23 +1,42 @@
 const express = require('express');
 const { getTopics } = require('./controllers/controllers.topics');
-const { patchArticle } = require('./controllers/controllers.articles');
+const {
+  patchArticle,
+  getArticle,
+} = require('./controllers/controllers.articles');
+const res = require('express/lib/response');
```
Comment (Rosem10) [inline app.js:7]: Don't need this line!

File: models/models.articles.js [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
+
+exports.selectArticle = async (article_id) => {
+  const results = await db.query(
+    `SELECT * FROM articles WHERE article_id = $1;`,
+    [article_id]
+  );
+  if (!results.rows.length) {
+    return Promise.reject({ status: 404, message: 'invalid article id' });
+  } else return results.rows[0];
+  console.log(results);
```
Comment (Rosem10) [inline models/models.articles.js:24]: Remove this console.log please!

**Review by Rosem10 — APPROVED** [review]
Really nice!

File: app.js (hunk 1 above)
Comment (abdullah7x) [reply app.js:7]: VScode keeps putting that in, I've no idea why. All cleaned up!

**Pull request merged**